lib/Doctrine/MongoDB/Aggregation/Expr.php

```diff
@@ -41,6 +42,23 @@ class Expr
     private $currentField;
 
     /**
+     * Returns the absolute value of a number.
+     *
+     * The <number> argument can be any valid expression as long as it resolves
+     * to a number.
+     *
+     * @see https://docs.mongodb.org/manual/reference/operator/aggregation/abs/
+     * @param mixed|self $number
```

Comment:
Is "self" just here to suggest Expr? At first, it looked redundant alongside "mixed", but I can understand the reasoning.